module/Photo/src/Photo/Service/Metadata.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Photo\Service;
+
+use Application\Service\AbstractService;
+use Zend\ServiceManager\ServiceManager;
+use Zend\ServiceManager\ServiceManagerAwareInterface;
+use Photo\Model\Album as AlbumModel;
+use Photo\Model\Photo as PhotoModel;
+
+/**
+ * Metadata service. This service implements all functionality related to 
+ * gathering metadata about photos.
+ */
+class Metadata extends AbstractService
+{
+
+    /**
+     * 
+     * @param \Photo\Model\Photo $photo the photo to add the metadata to
+     * @return \Photo\Model\Photo the photo with the added metadata
+     */
+    public function populateMetadata($photo, $path)
+    {
+        $exif = \read_exif_data($path, 'EXIF');
```

Comment:
Ik zie hem nu pas langskomen. De code ziet er uit alsof hij direct van de oude site gecopypaste is. Ik zal als ik tijd heb nog even kijken of ik de functies wat kan refactoren